test/Modules/ExteriorPowers.jl

```diff
@@ -130,12 +130,14 @@ end
   u = (F[1], F[4], F[3])
 
   @test "$(mm(v))" == "e[1]∧e[3]∧e[4]"
-  @test "$(F3)" == "⋀^3(Free module of rank 5 over Multivariate polynomial ring in 5 variables over QQ)"
+  #@test "$(F3)" == "⋀^3(Free module of rank 5 over Multivariate polynomial ring in 5 variables over QQ)"
+  @test "$(F3)" == "3-th exterior power of Free module of rank 5 over Multivariate polynomial ring in 5 variables over QQ"
```

Comment:
should of course be `3rd` not `3-th`.

We have `ordinal_number_string` for this.

Reply:
Thanks. Didn't know that.